1-exercises/A-this/exercise-1.js

```diff
@@ -15,7 +15,9 @@ let person = {
     name: "Adrian",
     age: 32,
     currentAddress: "London",
-    celebrateBirthday: /* todo */
+    celebrateBirthday: function (){
+        return (`Expected age is ${expectedAge}, actual age is ${this.age}`)
```

Comment:
The celebrateBirthday function should increment the age of the person object when called. This function shouldn't return anything, it should be similar to the deposit function in exercise-3 you implemented.

Reply:
this Maten, updated my changes.